.github/workflows/build_spdlog.yml

```diff
@@ -0,0 +1,193 @@
+    strategy:
+      matrix:
+        build_type: ['Release', 'Debug']
+        shared: ['True', 'False']
+    env:
+      CONAN_REVISIONS_ENABLED: 1
+    steps:
+    - uses: actions/checkout@v2
+    - name: install conan
+      run: pip install -U conan
+    - name: create profile
+      run: conan profile new tket --detect --force
+    - name: build spdlog
+      run: conan create --profile=tket -s build_type=${{ matrix.build_type }} -o spdlog:shared=${{ matrix.shared }} recipes/spdlog tket/stable
+    - name: add remote
+      run: conan remote add tket-conan https://tket.jfrog.io/artifactory/api/conan/tket-conan --force
+    - name: authenticate to repository
+      run: conan user -p ${{ secrets.JFROG_ARTIFACTORY_TOKEN_1 }} -r tket-conan ${{ secrets.JFROG_ARTIFACTORY_USER_1 }}
+    - name: get version
+      run: |
+        spdlog_ver=$(conan inspect --raw version recipes/spdlog/)
+        echo "SPDLOG_VER=${spdlog_ver}" >> $GITHUB_ENV
+    - name: upload package (dry run)
+      if: github.event_name == 'pull_request'
+      run: conan upload spdlog/${SPDLOG_VER}@tket/stable --all -r=tket-conan --skip-upload
+    - name: upload package
+      if: github.event_name == 'push'
+      run: conan upload spdlog/${SPDLOG_VER}@tket/stable --all -r=tket-conan
+  windows:
+    name: build spdlog (windows)
+    needs: changes
+    if: needs.changes.outputs.recipes_spdlog == 'true'
+    runs-on: windows-2019
+    strategy:
+      matrix:
+        build_type: ['Release', 'Debug']
+        shared: ['True', 'False']
+    env:
+      CONAN_REVISIONS_ENABLED: 1
+    steps:
+    - uses: actions/checkout@v2
+    - name: install conan
+      run: pip install conan
+    - name: create profile
+      run: conan profile new tket --detect
+    - name: normalize line endings in conanfile
+      run: |
+        $conanfile ='recipes/spdlog/conanfile.py'
+        $normalized_file = [IO.File]::ReadAllText($conanfile) -replace "`r`n", "`n"
+        [IO.File]::WriteAllText($conanfile, $normalized_file)
+    - name: build spdlog
+      run: conan create --profile=tket -s build_type=${{ matrix.build_type }} -o spdlog:shared=${{ matrix.shared }} recipes/spdlog tket/stable
+    - name: add remote
+      run: conan remote add tket-conan https://tket.jfrog.io/artifactory/api/conan/tket-conan
+    - name: authenticate to repository
+      run: conan user -p ${{ secrets.JFROG_ARTIFACTORY_TOKEN_1 }} -r tket-conan ${{ secrets.JFROG_ARTIFACTORY_USER_1 }}
+    - name: get version
+      run: |
+        $spdlog_ver = conan inspect --raw version recipes/spdlog/
+        echo "SPDLOG_VER=${spdlog_ver}" >> $env:GITHUB_ENV
+    - name: upload package (dry run)
+      if: github.event_name == 'pull_request'
+      run: conan upload spdlog/${{ env.SPDLOG_VER }}@tket/stable --all -r=tket-conan --skip-upload
+    - name: upload package
+      if: github.event_name == 'push'
+      run: conan upload spdlog/${{ env.SPDLOG_VER }}@tket/stable --all -r=tket-conan
+  manylinux:
```

Comment:
Why is the linux and the manylinux build needed?

Reply:
The manylinux build is for use by the release builds. It differs in that it uses a different libcpp.